runtime/src/lib.rs

```diff
@@ -64,6 +64,9 @@ pub type EventRecord = paint_system::EventRecord<Event, Hash>;
 
 pub mod registry;
 
+pub mod string32;
```

Comment:
No need to export the whole module
```suggestion
mod string32;
```

Reply:
This also silently exports trait instances, then, right?